lib/model/Product.dart

```diff
@@ -655,4 +666,28 @@ class Product extends JsonObject {
 // TODO (Optional) Add Nutri-Score disclaimers cf. https://github.com/openfoodfacts/openfoodfacts-dart/issues/193
     return result;
   }
+
+  @JsonKey(
+      name: 'no_nutrition_data',
+      toJson: JsonHelper.checkboxToJSON,
+      fromJson: JsonHelper.checkboxFromJSON)
+  bool get noNutritionData => _noNutritionData ?? _nutriments == null;
+
+  set noNutritionData(bool? noNutritionData) {
+    if (_noNutritionData == true) {
+      _nutriments = null;
+    }
+    _noNutritionData = noNutritionData;
+  }
+
+  @JsonKey(
+      name: 'nutriments', includeIfNull: false, toJson: Nutriments.toJsonHelper)
```

Comment:
Please put a `,` before the `)`.

Reply:
Ok, on other fields like `productNameInLanguages` the comma is also missing
Should I also add it to them?